scripts/bash_lib.sh

```diff
@@ -428,6 +433,9 @@ upgrade_test_type() {
         prev_major_version="$major.$((minor - 1))"
       fi
       ;;
+    "distro")
+      prev_major_version="0.0"
```

Comment:
Can't we use something more clear, like `NA` (not applicable). It's confusing IMO to have a number here.